install_config/web_console_customization.adoc

```diff
@@ -230,6 +230,33 @@ width and height. The ideal height is *20px*.
 Add the stylesheet as described in
 xref:loading-custom-scripts-and-stylesheets[Loading Extension Scripts and Stylesheets].
 
+[[changing-the-membership-whitelist]]
+== Customizing the Membership Whitelist
+
+The default whitelist in the membership page shows a subset of cluster roles, such as 
+`admin`, `basic-user`, `edit`, and so on. It also shows custom roles defined within a project.
+
+For example, to add your own set of custom cluster roles to the whitelist:
+
+[source, javascript]
+----
+MEMBERSHIP_WHITELIST: [
```

Comment:
Could we update it to be more clear then users could copy and configure more easily?
```
window.OPENSHIFT_CONSTANTS.MEMBERSHIP_WHITELIST = [
  "admin",
  "basic-user",
  "edit",
  "system:deployer",
  "system:image-builder",
  "system:image-puller",
  "system:image-pusher",
  "view",
  "custom-role-1",
  "custom-role-2"
];
```

Reply:
Testing has passed per [bug comment 8](https://bugzilla.redhat.com/show_bug.cgi?id=1577568#c8) and [bug comment 9](https://bugzilla.redhat.com/show_bug.cgi?id=1577568#c9)